examples/CONTRIBUTING.md

```diff
@@ -0,0 +1,109 @@
+# Contributing Guide
+
+The VMware Event Broker Appliance team welcomes contributions from the
+community.
+
+To help you get started making contributions to VMware Event Broker Appliance,
+we have collected some helpful best practices in the [Contributing
+guidelines](https://vmweventbroker.io/community#guidelines).
+
+
+## Working with examples
+
+- Create a topic branch in your fork repository where you want to base your work
+- Make the changes you need and commit them
+- Make sure your commit messages are in the proper format (see below) and signed
+- Push your changes to a topic branch in your fork of the repository
+- Submit a pull request
+
+> Before submitting a pull request, please make sure that your change satisfies the requirements specified [here](https://vmweventbroker.io/community#pull-requests)
+
+## Publish a new version of an example
+
+These examples are hosted in [`ghcr.io`](https://github.com/features/packages) and we will explain how to deploy the examples to this registry but you are free to use any registry of your like.
+
+To publish an image manually you will need a [personal github token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens) (Classic) and save it in a environment variable like `$GH_CLASSIC_TOKEN`.
+
+After you have your token in your environment you can login using
+
+```bash
+echo $GH_CLASSIC | ko login ghcr.io -u <your-username> --password-stdin
+```
+
+The rest of the steps depends on the tools use to build and push each example.
+
+## Go Examples
+
+Go-based examples use [`ko`](https://ko.build) to build and push images of the working directory of the example.
+
+For `ko`, you need to set the base url for the repository these images are going to be part of using the `KO_DOCKER_REPO`. We are going to use `ghcr.io/vmware-samples/vcenter-event-broker-appliance` but you can deploy an example to your fork.
+
+```bash
+export KO_DOCKER_REPO=ghcr.io/vmware-samples/vcenter-event-broker-appliance
+```
+
+You also need to set two environment variables for `ko`: `KO_TAG` and `KO_COMMIT`
+
+```bash
+export KO_COMMIT=$(git rev-parse --short=8 HEAD)
+export KO_TAG=<new-version>
+```
+
+Now you can build and push your new version of the example using `ko build`
```

Comment:
Since you use the `ghcr.io/vmware-samples/vcenter-event-broker-appliance` repo throughout this guide (besides in the Python section btw), normal users will see an error during push due to missing permissions

Reply:
I try to use a generic way to use the package repository that will be use but let me know if you would add or delete something

Reply:
You could just use `export KO_DOCKER_REPO=<your_docker_repo>`